### DISCUSSION TITLE
Is it possible to make a convenient and intuitive backend user management panel？

### DISCUSSION THREAD

**Discussion by linnene** [seapagan/fastapi-template#709] [Ideas]
**I haven't found a lightweight and open source user management framework. If there is a recommendation, I can give it a try and see if it can be integrated.**

**Comment by seapagan** [comment]
there is [fastapi-admin](https://github.com/fastapi-admin/fastapi-admin) though that may only work with TortoiseORM but [sqladmin](https://github.com/aminalaee/sqladmin) may be able to be integrated? I've not looked at them fully but may be worth a punt.

**Comment by seapagan** [comment]
Working on adding [sqladmin](https://github.com/aminalaee/sqladmin) as an admin panel. Seems to be quite good. I'm tweaking it, but I also need to integrate authentication before releasing it.

![image](https://github.com/user-attachments/assets/be7e7564-65ce-45fe-acce-9614aee96bff)

**Reply by seapagan** [reply]
track this on PR #738

**Reply by seapagan** [reply]
Now merged into main :partying_face: 

Try it out and let me know any issues or requests

**Reply by linnene** [reply]
It's incredible, I thought this proposal had fallen into the air, but I didn't expect you to actually try it, and I will use it as soon as I finish the project at hand and report it to you

**Reply by seapagan** [reply]
Excellent :grin: 

I was working on the project again and got inspired to add that so hope it's useful. TBH, the hard part was adding the test coverage :rofl: 

Edit: Look at the [docs site](https://api-template.seapagan.net/usage/admin-panel/) for info on how to enable and use.

Let me know if you have any issues or ideas to improve.
